lib/screens/finder.rb

```diff
@@ -55,12 +55,12 @@ def ask_to_expand
   end
 
   def expand_contact(matched)
-    index = ask_for_contact - 1
+    index = ask_for_contact_index - 1
```

Comment:
It's because arrays start at 0. I could rewrite the DB class to handle indexes that start at 1 instead.